public/app/core/components/NavBar/NavBarNext.tsx

```diff
@@ -47,13 +47,24 @@ export const NavBarNextUnconnected = React.memo(({ navBarTree }: Props) => {
     setShowSwitcherModal(!showSwitcherModal);
   };
   const navTree = cloneDeep(navBarTree);
-  const coreItems = navTree.filter((item) => item.section === NavSection.Core);
+  const coreItems = navTree.filter((item) => {
+    const shownItemIds = ['home', 'dashboards'];
+    return item.section === NavSection.Core && shownItemIds.indexOf(item.id as string) !== -1;
+  });
   const pluginItems = navTree.filter((item) => item.section === NavSection.Plugin);
-  const configItems = enrichConfigItems(
+  let configItems = enrichConfigItems(
     navTree.filter((item) => item.section === NavSection.Config),
     location,
     toggleSwitcherModal
   );
+  configItems = configItems.filter((item) => {
```

Comment:
Based on the discussion earlier today, we can just completely remove the configItems section right?

Reply:
We're going to attempt to upstream a configuration for this change instead.